Dockerfile

```diff
@@ -23,6 +23,7 @@ FROM registry.svc.ci.openshift.org/ocp/4.7:base
 RUN INSTALL_PKGS=" \
       openssl \
       rsync \
+      xz \
```

Comment:
Do we know if this is available downstream? Consider asking ART

Reply:
It is. The downstream base image openshift-ose-base:v4.6.0.20201030.115241 is based on UBI 8.2, which in turn does provide the xz package according to https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?container-tabs=packages .